src/tool/core/helpers/problem_code.rs

```diff
@@ -354,7 +369,7 @@ impl TryFrom<&str> for FunctionArgType {
             "Option<Box<ListNode>>" => List,
             "Option<Rc<RefCell<TreeNode>>>" => Tree,
             trimmed_value => {
-                warn!("Unknown type \"{trimmed_value}\" found please report this in an issue https://github.com/rust-practice/cargo-leet/issues/new");
+                warn!("Unknown type {trimmed_value:?} found please report this in an issue https://github.com/rust-practice/cargo-leet/issues/");
```

Comment:
can we add `?template=missing_type.md`  to the link ?

Reply:
Added